src/index.ts

```diff
@@ -75,15 +76,6 @@ export const toTitleCase = (text: string) => {
     const lowercaseWord = word.toLowerCase()
     counter += 1
 
-    // Lowercase "x" in "Artsy X Capsule auctions" case
-    if (
-      lowercaseWord === "x" &&
-      isSpaceCharacter(text[index - 1]) &&
-      isSpaceCharacter(text[index + word.length])
-    ) {
-      return lowercaseWord
-    }
-
```

Comment:
`x` must remain in lowercase (you can learn more [here](https://artsyproduct.atlassian.net/browse/FX-3198?focusedCommentId=40173))

Reply:
note that "x" was added to `alwaysLowercaseWords`

Reply:
any updated on this? `x` remains lowercase with the changes